### ISSUE TITLE
onnx simplifier

### ISSUE THREAD

**Issue by chrisgank** [daquexian/onnx-simplifier#175]
你好我在使用onnxsim指令时出现了错误，如下
onnxruntime.capi.onnxruntime_pybind11_state.RuntimeException: [ONNXRuntimeError] : 6 : RUNTIME_EXCEPTION : Exception during initialization: D:\a\_work\1\s\onnxruntime\core/providers/cpu/nn/batch_norm.h:52 onnxruntime::BatchNorm<floa
t>::BatchNorm !is_train_ was false. Training mode is not supported in this build.
将onnx模型简化的时候出现了错误，希望能解答，谢谢。
